ddtrace/tracer/span.go

```diff
@@ -408,6 +408,21 @@ func (s *Span) setSamplingPriorityLocked(priority int, sampler samplernames.Samp
 	s.context.setSamplingPriority(priority, sampler)
 }
 
+// forceSamplingPriorityLocked updates the sampling priority.
+// If the trace is locked, the sampling priority is forced to the given value.
+//
+// This function is should only be used when applying a manual keep or drop decision.
+func (s *Span) forceSamplingPriorityLocked(priority int, sampler samplernames.SamplerName) {
```

Comment:
super nit: to align with the existing (slew of) sampling priority functions we have, maybe we can name this one `forceSetSamplingPriorityLocked` and call `forceSetSamplingPriority`. Without the `set` it makes me think these are getter functions instead of setters